### ISSUE TITLE
Strings on continued lines not formatted correctly

### ISSUE THREAD

**Issue by mhammack** [dkelosky/vscode-ibm-hlasm#13]
Noticed that when a string is continued on the next line (no closing quote on the first line), the highlighting for both lines is off:

![Capture 12-1](https://user-images.githubusercontent.com/20606553/62495266-fc76c700-b79a-11e9-8283-84f1154679dc.PNG)
(the space in column 16 of the second DC is part of the string).

**Comment by CoderCoco** [comment]
Yeah this is gonna be tricky to fix. One other thing that you might see with strings is something where you'd have `L'...L'`, the string highlighting takes over the `L'` highlighting.

Also to address this and keep the continuation character intact is going to be a bit tricky due to the regex limitations.

**Comment by dkelosky** [comment]
@mheuzey - PRs welcome 😉
